frontend/src/metabase/components/SaveQuestionForm/types.ts

```diff
@@ -9,12 +9,37 @@ export type SaveQuestionProps = {
 
   closeOnSuccess?: boolean;
   multiStep?: boolean;
-  initialCollectionId?: CollectionId | null;
+
+  /**
+   * The target collection to save the question to.
+   * Currently used for the embedding SDK.
+   *
+   * When this is defined, the collection picker will be hidden and
+   * the question will be saved to this collection.
+   ***/
```

Comment:
One extra `*` can be dropped 😉 to stay consistent with the rest of the comments

```suggestion
   **/
```

Reply:
Good eye! I didn't notice that